mobilizon_reshare/event/event_selection_strategies.py

```diff
@@ -36,7 +41,7 @@ def _select(
         self,
         published_events: List[MobilizonEvent],
         unpublished_events: List[MobilizonEvent],
-    ) -> Optional[MobilizonEvent]:
+    ) -> Optional[List[MobilizonEvent]]:
```

Comment:
I think the added value is the order of the publications, but I agree we can avoid changing the strategy logic. Maybe it could be worth to have a separate command such as `mobilizon-reshare plan` so that you could see the planned order of publication and maybe be able to change it.